tensorboard/plugins/profile/profile_demo.op_profile.json

```diff
@@ -0,0 +1,95 @@
+{
```

Comment:
Is this expected? I see nothing in the parentheses.

![knxthhfajij](https://user-images.githubusercontent.com/4221553/30204057-fc0297f4-9438-11e7-9b2a-4af11d409fce.png)

Reply:
Also, "is overhead, and a good target for optimization" is a dangling half-sentence. I think something might be missing there.